package/yast2-auth-client.changes

```diff
@@ -1,4 +1,15 @@
 -------------------------------------------------------------------
+Mon Jan  5 17:51:59 UTC 2015 - varkoly@suse.com
+
+- bnc#893983 - yast2-auth-client has hard requires for sssd
```

Comment:
The message does not make it clear whether the hard requires is the buggy state or the fixed state. IMHO better: "Removed an unnecessary hard dependency on sssd."